meta-sokol-flex-support/recipes-core/meta/codebench-makefile.bb

```diff
@@ -0,0 +1,38 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: MIT
+# ---------------------------------------------------------------------------------------------------------------------
+
+SUMMARY = "ADE Makefile Fragment for CodeBench"
+LICENSE = "MIT"
+INHIBIT_DEFAULT_DEPS = "1"
+
+S = "${WORKDIR}"
+
+ADE_PRE_BUILD_TARGET ?= ""
```

Comment:
Will we still use ADE in our meta data? I can see other variables also have ADE in it. Are we planning to update them?

Reply:
I think this one was an oversight, thanks!